Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -69,7 +69,7 @@ public final class DependencyTreeGenerator {
 
     // MARK: - DependencyTreeGeneratorError
 
-    enum DependencyTreeGeneratorError: Error, CustomStringConvertible {
+    private enum DependencyTreeGeneratorError: Error, CustomStringConvertible {
```

Comment:
not a necessary change but noticed that I'd forgotten to make this `private` while I was here.